src/BotCatMaxy/Components/Moderation/Settings.cs

```diff
@@ -45,6 +46,27 @@ public async Task<RuntimeResult> ToggleServerIDUse()
                 "This is a legacy feature, if you want this done now contact blackcatmaxy@gmail.com with your guild invite and your username so I can get back to you");
         }
 
+        [Command("setappeallink"), Alias("appeallink")]
+        [Summary("Sets the link sent to all banned users to appeal. Use this command by itself to disable.")]
+        [RequireContext(ContextType.Guild)]
+        [HasAdmin]
+        public async Task SetAppealLink(string link = "")
+        {
+            ModerationSettings settings = Context.Guild.LoadFromFile<ModerationSettings>(true);
+
+            if (link.IsNullOrEmpty())
+                settings.appealLink = null;
+            else
+                settings.appealLink = link;
+
+            settings.SaveToFile();
+
+            if (link.IsNullOrEmpty() || link == "none" || link == "null")
```

Comment:
Want to use .`ToLower()` so it doesn't complain when a user randomly capitalizes it (look for bad example in `!setpubchannel` in `LoggingCommands`

Reply:
Should I change it for the other settings? (I looked at your older code for reference whether I should do that or not.)

Reply:
We should keep this PR consistent with what we can and do another pass later